src/attributes.jl

```diff
@@ -855,15 +855,23 @@ The values indicate how to interpret the result vector.
   infeasibility.
 * `NEARLY_INFEASIBILITY_CERTIFICATE`: the result satisfies a relaxed criterion for
   a certificate of infeasibility.
+* `REDUCTION_CERTIFICATE`: the result vector is an illposed certificate. If the
```

Comment:
This is a pretty esoteric status. Could you add a link to a page/paper that describes what it means?